pkgs/by-name/xi/xivlauncher/package.nix

```diff
@@ -1,19 +1,35 @@
 { lib, buildDotnetModule, fetchFromGitHub, dotnetCorePackages, SDL2, libsecret, glib, gnutls, aria2, steam, gst_all_1
 , copyDesktopItems, makeDesktopItem, makeWrapper
-, useSteamRun ? true }:
+, useSteamRun ? true
+, useGameMode ? false
+, useRbPatchedLauncher ? false }:
 
 let
-  rev = "1.0.8";
+  goatcorp = {
+    version = "1.0.8";
+    owner = "goatcorp";
+    rev = "1.0.8";
```

Comment:
Given that `version` and `rev` are the same in upstream XIVLauncher, is there a neater way to do this?